webpack.config.babel.js

```diff
@@ -0,0 +1,26 @@
+const CompressionPlugin = require('compression-webpack-plugin');
+const { rules, plugins } = require('webpack-atoms');
```

Comment:
change back to `webpack.config.js`?

Reply:
Sure. But config for rollup driven builds is different. Ofcouse a single config file with some env variable or some other trick can help us identify which bundler is being used, but focus of the PR is to compare the two build systems. Things are not perfect :)